ubuntu1604/profiles/standard.profile

```diff
@@ -1,10 +1,10 @@
 documentation_complete: true
 
-title: 'Standard System Security Profile for Ubuntu 16'
+title: 'Standard System Security Profile for Ubuntu 16.04'
```

Comment:
@cipherboy since we are fixing this content now... remove the `.04`. We don't keep track of minor versions.... and neither do we want to.